test/fixtures/schema/default-sample.json

```diff
@@ -0,0 +1,152 @@
+                                          "content": "list"
+                                        },
+                                        "value": {
+                                          "element": "array",
+                                          "attributes": {
+                                            "default": [
+                                              {
+                                                "element": "string",
+                                                "content": "4"
+                                              }
+                                            ]
+                                          },
+                                          "content": [
+                                            {
+                                              "element": "string",
+                                              "content": "3"
+                                            },
+                                            {
+                                              "element": "string",
+                                              "content": "2"
+                                            },
+                                            {
+                                              "element": "string",
+                                              "content": "1"
+                                            }
+                                          ]
+                                        }
+                                      }
+                                    }
+                                  ]
+                                }
+                              ]
+                            },
+                            {
+                              "element": "asset",
+                              "meta": {
+                                "classes": [
+                                  "messageBody"
+                                ]
+                              },
+                              "attributes": {
+                                "contentType": "application/json"
+                              },
+                              "content": "{\n  \"list\": [\n    \"3\",\n    \"2\",\n    \"1\"\n  ]\n}"
+                            },
+                            {
+                              "element": "asset",
+                              "meta": {
+                                "classes": [
+                                  "messageBodySchema"
+                                ]
+                              },
+                              "attributes": {
+                                "contentType": "application/json"
+                              },
+                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"list\": {\n      \"type\": \"array\",\n      \"default\": \"4\"\n    }\n  }\n}"
```

Comment:
This needs a closer look. How can an array have a default value of `4`? Isn't `4` a primitive?